microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension/get-started.md

```diff
@@ -0,0 +1,366 @@
+
+   Otherwise you might get an error message because the folder that contains the Success page is not trusted, and CSS mirror editing is trying to edit the CSS in the `index.html` source file in that folder.
+
+1. In the **Edge DevTools: Browser** tab, in the address bar, select and copy the file path, but not the file name, such as `C:/Users/myusername/.vscode/extensions/ms-edgedevtools.vscode-edge-devtools-2.1.1/out/startpage/`.
+
+1. In Activity Bar > **Explorer**, click the **Open Folder** button.  In the **Open Folder** dialog, paste or select the path you copied above.  To paste, on Windows, you might need to change `/` to `\` throughout the path. Then click the **Select Folder** button.
+
+   The first time you open a folder, you must confirm that you trust the authors of files in this folder:
+
+   ![Do you trust the authors in the files of this folder?](./get-started-images/trust.png)
+
+1. Click the **Yes, I trust the authors** button.
+
+   You might need to launch DevTools again, as follows.
+
+1. In Activity Bar, click **Microsoft Edge Tools** (![Microsoft Edge Tools button](./get-started-images/microsoft-edge-tools-icon.png)).  The **Microsoft Edge Tools** Side Bar opens.
+
+1. Click the **Launch Instance** button.
+
+   The **Edge DevTools** tab opens, and the **Edge DevTools: Browser** tab opens, displaying the default, Success page.  The Success page source file is a self-contained `.html` file in a directory on your drive.  This is a single `.html` file that includes CSS rules (in a `<style>` element).  It also includes a JavaScript `console` statement (in a `<script>` element).
+
+   **Edit CSS:**
+
+1. In the **Edge DevTools** tab, click the **Select an element in the page to inspect it** (![Inspect tool icon](./open-devtools-and-embedded-browser-images/inspect-tool-icon.png)) button, sometimes called the **Inspect** button.
+
+1. In the **Edge DevTools: Browser** tab, hover over different parts of the page, while you watch the **Elements** tool's DOM tree expand and update.
+
+1. Click the **Success!** heading, which is an `<h2>` element.
+
+1. In the **Styles** tab of the **Elements** tool in the **Edge DevTools** tab, in the non-italic h2 declaration section, click to the right of the `margin-bottom` rule.
+
+1. Enter a new CSS rule, `font-size: 5em`, and then press `Enter`.  The spelling is similar to the rule shown in the italic h2 section from the user agent stylesheet below it.
+
+   `index.html` opens, automatically edited by CSS mirror editing to add the line `font-size: 5em;`.  Scroll to that line, in the h2 section.
+
+  **Arrange tabs:**
```

Comment:
This bolded section title isn't indented like the other ones, and therefore appears left-aligned in the rendered page, unlike the other ones. And the numbered list items after it re-start at 1.

Reply:
indented